e2e/exec.go

```diff
@@ -55,12 +55,7 @@ func (r podExecResult) dumpStderr() {
 	}
 }
 
-func extractLogsFromVM(ctx context.Context, t *testing.T, vmssName string, sshPrivateKey string, opts *scenarioRunOpts) (map[string]string, error) {
-	privateIP, err := getVMPrivateIPAddress(ctx, opts.cloud, opts.suiteConfig.subscription, *opts.chosenCluster.Properties.NodeResourceGroup, vmssName)
-	if err != nil {
-		return nil, fmt.Errorf("unable to get private IP address of VM on VMSS %q: %w", vmssName, err)
-	}
-
+func extractLogsFromVM(ctx context.Context, t *testing.T, vmssName string, privateIP, sshPrivateKey string, opts *scenarioRunOpts) (map[string]string, error) {
```

Comment:
privateIP needs the string type for consistency's sake, or we can remove the string type specifier from vmssName

Reply:
good catch, thanks